css/css-conditional/test_group_insertRule.html

```diff
@@ -34,6 +35,8 @@
     test(function() {
            assert_equals(grouping_rule.type, rule_types[i],
                          "Rule type of " + rule_name + " rule");
+           assert_equals(grouping_rule.type, rule_nums[i],
```

Comment:
It would be better if this assert was in a separate 
```
test(function() {
    assert_equals(…);
    },
   rule_name + " rule number");
```

Rather than grouped with the "rule type" function. Tests results will be the same, but the info in the log will be clearer.